recipe/build.sh

```diff
@@ -10,3 +9,16 @@ bash configure  --prefix=$PREFIX $OPTS
 make
 make check
 make install
+
+if [ "$(uname)" == "Darwin" ]; then
+    # Based on
+    # https://github.com/Homebrew/homebrew-core/blob/master/Formula/emacs.rb
+    mv nextstep/Emacs.app $PREFIX/Emacs.app
```

Comment:
No, I have to do that (that's what's causing so much headache for me in this PR).